geoserver/geoserver_data/global/dev/gwc-gs.xml

```diff
@@ -0,0 +1,43 @@
+<GeoServerGWCConfig>
+    <version>1.1.0</version>
+    <directWMSIntegrationEnabled>true</directWMSIntegrationEnabled>
+    <requireTiledParameter>true</requireTiledParameter>
+    <WMSCEnabled>true</WMSCEnabled>
+    <TMSEnabled>true</TMSEnabled>
+    <securityEnabled>false</securityEnabled>
+    <innerCachingEnabled>false</innerCachingEnabled>
+    <persistenceEnabled>true</persistenceEnabled>
+    <cacheProviderClass>class org.geowebcache.storage.blobstore.memory.guava.GuavaCacheProvider</cacheProviderClass>
+    <cacheConfigurations>
+        <entry>
+            <string>class org.geowebcache.storage.blobstore.memory.guava.GuavaCacheProvider</string>
+            <InnerCacheConfiguration>
+                <hardMemoryLimit>16</hardMemoryLimit>
+                <policy>NULL</policy>
+                <concurrencyLevel>4</concurrencyLevel>
+                <evictionTime>120</evictionTime>
+            </InnerCacheConfiguration>
+        </entry>
+    </cacheConfigurations>
+    <cacheLayersByDefault>true</cacheLayersByDefault>
+    <cacheNonDefaultStyles>true</cacheNonDefaultStyles>
+    <metaTilingX>4</metaTilingX>
+    <metaTilingY>4</metaTilingY>
+    <gutter>0</gutter>
+    <defaultCachingGridSetIds>
+        <string>EPSG:4326</string>
```

Comment:
Maybe check with Admire if its here that we need to add EPSG:3857 ?

Reply:
Yes, you will need to define the defaults here as well. I.e
```
<defaultCachingGridSetIds>
    <string>EPSG:3857</string>
    <string>EPSG:4326</string>
    <string>EPSG:900913</string>
  </defaultCachingGridSetIds>
```
Provided the `EPSG:3857` is defined in `geowebcache.xml`

Reply:
EPSG:3857 added as a default. Please check.